generated/nixnet/nixnet.proto

```diff
@@ -1103,10 +1103,11 @@ message ConnectTerminalsResponse {
 message ConvertByteArrayToFramesSinglePointRequest {
   nidevice_grpc.Session session_ref = 1;
   bytes value_buffer = 2;
-  uint32 size_of_buffer = 3;
+  uint32 num_of_frames = 3;
```

Comment:
nit: we used `number_of_signals` for #619 so I would use `number_of_frames` here.

Also, for some reason `num_frames` sounds fine/normal to me but `num_of_frames` sounds weird.

Reply:
Updated to `number_of_frames` for consistency.